model/model.go

```diff
@@ -371,10 +371,10 @@ type DestinationSpanContext struct {
 // DestinationServiceSpanContext holds contextual information about a
 // destination service.
 type DestinationServiceSpanContext struct {
-	// Type holds the destination service type.
+	// Type holds the destination service type. Deprecated.
 	Type string `json:"type,omitempty"`
 
-	// Name holds the destination service name.
+	// Name holds the destination service name. Deprecated.
 	Name string `json:"name,omitempty"`
```

Comment:
I forget whether we treat "" the same as omitted in apm-server, but maybe we can just remove "omitempty" from these two fields so we always send them as empty strings for backwards compatibility with older server versions that require the fields